### PR TITLE
feat(blog): Blog Post Layout and Blog Post

### LINKED ISSUE #20: Task: Blog Page layout
**Describe the new feature**
Let's add the following contents to the home page (`unixsocial.com/blog`)

Our blog page will render a Blog layout

1. Add logo on the left
2. Blog Search #33 is a _separate issue in isolation_
3. Flexible left hand menu area
4. Flexible Body area

**Where the feature should be placed**
In the blog feature on a separate page

**Sketches**
Video description has been provided via Telegram

**Additional context**
All dependencies has already resolved:
1. #13 
2. #14
3. #24

### PR DESCRIPTION
**Pull request by AhsenRiaz**
fix #20 #30

### CODE REVIEW

**Comment by attilacsanyi** [comment]
@AhsenRiaz be sure to mark PR for review only if CI is green so `pnpm run ci` locally running at least. Let me know if you have any issue you cannot resolve. I see linting already failed and some issues can be autofixed if you run the above command. Thanks for the collaboration.

**Comment by AhsenRiaz** [comment]
@attilacsanyi You might notice that I have removed the export of **CardTitle** component in ShadCn Card Module. The reason is that it's giving an unexpected lint warning due to which **CI** kept failing, even if don't use the **CardTitle** component. It also kept giving this error in the CI-CD pipeline but after removing the export for the **CardTitle** CI-CD ran successfully.

This is the screenshot of the 
lint

I might be wrong so you can also check it on your end to let me know more about it 👍

**Comment by attilacsanyi** [comment]
Please remove the duplicated menu and add link back to Landing page if user clicking on Home if @uniscinft agree:

**Comment by attilacsanyi** [comment]
Please use the same spacing on the marked places below _(left-right, top-bottom should be the same value)_ it is visible:

**Comment by attilacsanyi** [comment]
Check with @uniscinft you might remember he mentioned the **missing comment button**.

File: libs/ui/src/lib/blog-layout/blog-search.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export const BlogSearch = () => {
+  return (
+    <div className="relative flex w-full">
+      <Input
+        type="text"
+        placeholder="Search Here"
+        className="placeholder w-full rounded-lg bg-secondary p-2"
+      />
+      <div className="absolute right-1 top-1/2 -translate-y-1/2 cursor-pointer rounded-sm p-4 ">
+        <SearchIcon />
```
Comment (attilacsanyi) [inline libs/ui/src/lib/blog-layout/blog-search.tsx:13]: Please use consistent icon sizes providing the same `className="w-5 h-5"` as in other icon cases.

File: libs/ui/src/lib/blog-layout/blog-post.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      timestamp: '6:34 AM',
+      content:
+        '🌐Welcome to UniX - Where the future of social meets Web3! ✨A space where privacy is paramount, transparency is key and you are in control! 🔗Stay tuned for exciting updates and join us as we shape the future of social media: http://t.me/unixsocial',
+    },
+  ];
+
+  return blogPosts.map((blogPost, i) => {
+    const { avatar, name, timestamp, date, content } = blogPost;
+    return (
+      <Card key={i} className=" w-full rounded-sm bg-secondary">
```
Comment (attilacsanyi) [inline libs/ui/src/lib/blog-layout/blog-post.tsx:28]: `bg-secondary` not needed as we discussed

File: libs/ui/src/lib/blog-layout/blog-search.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { SearchIcon } from 'lucide-react';
+import { Input } from '../shadcn-ui';
+
+export const BlogSearch = () => {
+  return (
+    <div className="relative flex w-full">
+      <Input
+        type="text"
+        placeholder="Search Here"
+        className="placeholder w-full rounded-lg bg-secondary p-2"
```
Comment (attilacsanyi) [inline libs/ui/src/lib/blog-layout/blog-search.tsx:10]: `bg-secondary` not needed as we discussed

**Comment by attilacsanyi** [comment]
On a big screen it should be responsive. Check how it is layout just on a mac screen which is not considered large:

1. please use `grid` layout, 2 or 3 blogpost can fit into one row as screen changes
1. search bar can  fill max half of the available width above the blog grid
1. mobile first is good each blog under each other 👌

**Comment by attilacsanyi** [comment]
> @attilacsanyi You might notice that I have removed the export of **CardTitle** component in ShadCn Card Module. The reason is that it's giving an unexpected lint warning due to which **CI** kept failing, even if don't use the **CardTitle** component. It also kept giving this error in the CI-CD pipeline but after removing the export for the **CardTitle** CI-CD ran successfully.
> 
> This is the screenshot of the warning lint
> 
> I might be wrong so you can also check it on your end to let me know more about it 👍

Thanks for reporting, but feel free to use ignoring eslint rules as we use the genereated code from ChadCN UI.
I've already did that find this code: `// eslint-disable-next-line jsx-a11y/heading-has-content`

**Comment by AhsenRiaz** [comment]
> On a big screen it should be responsive. Check how it is layout just on a mac screen which is not considered large:
> 
> 1. please use `grid` layout, 2 or 3 blogpost can fit into one row as screen changes
> 2. search bar can  fill max half of the available width above the blog grid
> 3. mobile first is good each blog under each other 👌
> 
>

**Review by attilacsanyi — APPROVED** [review]
I think will be fine I applied some tweaks as an extra, if you like it can be merged, thx

**Comment by AhsenRiaz** [comment]
Thanks for the help, time to merge it now.

**Pull request merged**